engine/src/main/java/org/terasology/engine/subsystem/rpc/DiscordRPCSubSystem.java

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.engine.subsystem.rpc;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import com.jagrosh.discordipc.IPCClient;
+import com.jagrosh.discordipc.IPCListener;
+import com.jagrosh.discordipc.entities.RichPresence;
+import com.jagrosh.discordipc.entities.pipe.Pipe;
+import com.jagrosh.discordipc.entities.pipe.WindowsPipe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.config.Config;
+import org.terasology.context.Context;
+import org.terasology.engine.GameEngine;
+import org.terasology.engine.subsystem.EngineSubsystem;
+
+import java.time.OffsetDateTime;
+
+/**
+ * It's a subsystem that manages the discord rpc in the game client. It manages the status, connection and other stuff.
```

Comment:
Trivial writing style advice: You don't need the `It's` here and other minor streamlining

```suggestion
 * Subsystem that manages Discord RPC in the game client, such as status or connection.
```

Reply:
**Done** It's a new lesson I learned it today. Thank you 😃